artworkSubmitter.js

```diff
@@ -0,0 +1,530 @@
+const itemData = {
+	r: require('./itemdata/radicals.json'),
+	k: require('./itemdata/kanji.json'),
+	v: require('./itemdata/vocab.json'),
+};
+
+const itemNames = {
+	r: 'Radical',
+	k: 'Kanji',
+	v: 'Vocab',
+};
+
+const artworks = {
+	6: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/5/6/f/56f448502c1e93a871d9c6c47bb1f9a23950b531.png", "Barb", "radical"],

```

Comment:
I'm okay with having this in the repo since it's static but please move this massive array to a .json file so it doesn't overshadow the rest of the code in this file.